test/index.js

```diff
@@ -556,6 +556,7 @@ describe('Metalsmith', function(){
       fs.mkdirSync(fixture('build/build'))
       exec('touch test/fixtures/build/build/empty.md', function(err){
         if (err) return done(err)
+        // eslint-disable-next-line
         var files = { 'index.md': { contents: new Buffer('body') }}
```

Comment:
Are these lines necessary? I think the lint warning is right; this variable isn't being used for anything. It seems like an unnecessary bit of code that was copy-pasted from other test cases where it actually was needed.